docs/src/main/asciidoc/_cloud/Getting_Started.adoc

```diff
@@ -0,0 +1,156 @@
+```
+Expected output:
+```
+todo-backend-someuser-dev.apps.sandbox.x8i5.p1.openshiftapps.com
+```
+You should now be able to access your application using, for example
+```
+https://todo-backend-someuser-dev.apps.sandbox.x8i5.p1.openshiftapps.com
+```
+You can use the todo-frontend to test out the deployed app using:
+```
+https://todobackend.com/client/index.html
+```
+and entering your application URL from above.
+
+If you wish to remove the application when you are done with it, simply run:
+```
+$ helm delete todo-backend
+```
+
+## Example: jaxrs-client
+The second example to build and deploy to OpenShift is the
+https://github.com/wildfly/quickstart/tree/main/jaxrs-client[jaxrs-client] quickstart.
+
+First of all, clone the quickstart repo:
+```
+$ git clone https://github.com/wildfly/quickstart.git
+cd quickstart/jaxrs-client
+```
+
+WildFly Helm Charts is used to build and deploy `jaxrs-client`. The following command installs a Helm Chart in a OpenShift cluster (users can connect to their sandbox following the instructions already provided above):
+```
+$ helm install jaxrs-client-from-chart -f charts/helm.yaml wildfly/wildfly
+```
+
+This time, the name of the Helm Chart app is `jaxrs-client-from-chart` (of course, this may be changed to a different name, if desired)
+
+The application will now begin to build, and can be observed via:
+```
+$ oc get build -w
+```
+Once the build is complete, to follow the deployment of the application, run:
+```
+$ oc get deployment jaxrs-client-from-chart -w
+```
+Once deployment is complete, we need to define a route to expose the application on the internet. To do so, run:
+```
+$ oc get route jaxrs-client-from-chart -o jsonpath="{.spec.host}"
+```
+The above command returns back a public address that points to the application deployed to the OpenShift cluster of the user (for example):
+```
+jaxrs-client-from-chart-someuser-dev.apps.sandbox.x8i5.p1.openshiftapps.com
+```
+At this point, defining a env variable for simplicity, it is possible to test the REST endpoint that has been deployed into the OpenShift cluster:
+```
+$ export ROUTE=$(oc get route jaxrs-client-from-chart -o jsonpath="{.spec.host}")
+```
+and use curl to send HTTP method requests to the REST endpoint, for example (GET)
+```
+$ curl -L $ROUTE/rest/contacts
+```
+To run the quickstart's Arquillian integration tests to validate that the quickstart has been set up correctly
+```
+mvn clean verify -Parq-remote -Dserver.host=https://$(oc get route jaxrs-client-from-chart --template='{{ .spec.host }}')
+```
+
+Take a look at the `wildfly-maven-plugin` configuration in the `jaxrs-client` quickstart's https://github.com/wildfly/quickstart/blob/main/jaxrs-client/pom.xml#L274-L299[pom.xml]. Notice that it specifies the layer that should be used when provisioning WildFly. For more details about this, take a look at the `wildfly-maven-plugin` https://docs.wildfly.org/wildfly-maven-plugin/[documentation].
```

Comment:
Let's update the line numbers to: https://github.com/wildfly/quickstart/blob/main/jaxrs-client/pom.xml#L270-L302 and then reword this as follows:

Take a look at the `openshift` profile configuration in the `jaxrs-client` quickstart's https://github.com/wildfly/quickstart/blob/main/jaxrs-client/pom.xml#L270-L302[pom.xml]. The Maven profile named `openshift` is used by the Helm chart to provision the server with the quickstart deployed. Notice the `wildfly-maven-plugin` configuration defined in this profile. It specifies the layer that should be used when provisioning WildFly. For more details about this, take a look at the `wildfly-maven-plugin` https://docs.wildfly.org/wildfly-maven-plugin/[documentation].